src/DayPicker.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { useCallback, useMemo } from "react";
 import type { MouseEvent, FocusEvent, KeyboardEvent, ChangeEvent } from "react";
 
+import { flushSync } from "react-dom";
```

Comment:
I think we can't use this function as it would require users to have latest `react-dom` - also from the docs it looks like we can skip it?